src/project.rs

```diff
@@ -93,7 +95,11 @@ impl Project {
 
     pub fn get_keyname(&self) -> String {
         if self.keyname.is_empty() {
-            self.prefix.clone()
+            if self.reuse_private_key.unwrap_or(false) {
+                self.prefix.clone()
+            } else {
+                format!("{}_{}", &self.prefix, &self.version.clone().unwrap())
```

Comment:
`unwrap` is added a few times in this PR, make sure `unwrap_or_print` is used wherever allowed. That is what prints the nice readable error messages rather.